test/unit/server/routes/health.spec.js

```diff
@@ -22,23 +28,33 @@ describe('GET /health', () => {
       }),
   );
 
-  context('when the build-info.json file is present', () => {
-    const projectRoot = path.resolve(__dirname, '../../../../');
-    const buildJson = path.resolve(projectRoot, 'build-info.json');
+  context('when the DB isn\'t working', () => {
```

Comment:
for this specific case the behavior you are testing is a file being present in the file system and being displayed as part of the response. I think an integration test would be more appropriate as it cover the real world usage. But I that there is more involved to produce the this JSON so perhaps I'm over thinking this